### PR TITLE
Add sysfsutils package

### PR DESCRIPTION
**Pull request by JL2210**
This package's purpose is to provide a set of utilities for interfacing
with sysfs, a virtual filesystem in Linux kernel versions 2.5+ that
provides a tree of system devices. While a filesystem is a very useful
interface, we've decided to provide a stable programming interface
that will hopefully make it easier for applications to query system devices
and their attributes.

Tested on x86_64

### CODE REVIEW

**Comment by uberhacker** [comment]
Do any Chromebooks use this filesystem?  I guess my question is why do we need this?

**Comment by JL2210** [comment]
Of course. Ever look at `/sys`? It's a dependency for another package.

**Comment by uberhacker** [comment]
What is the "other" package that needs this dependency?

**Comment by JL2210** [comment]
`rng_tools`, a program for gathering random data from device drivers and ahrdware random number generators. It uses this to access the `*random` devices in `/dev`. The `rng_tools` package is useful for generating GPG keys.

File: packages/sysfsutils.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Sysfsutils < Package
+  description 'Libsysfs provides a stable programming interface to sysfs and eases querying system devices and their attributes.'
```
Comment (uberhacker) [inline packages/sysfsutils.rb]: Shouldn't this description mention sysfsutils instead of libsysfs?

File: packages/sysfsutils.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  homepage 'https://github.com/Distrotech/sysfsutils'
+  version '2.1.0'
+  source_url 'http://http.debian.net/debian/pool/main/s/sysfsutils/sysfsutils_2.1.0+repack.orig.tar.gz'
```
Comment (uberhacker) [inline packages/sysfsutils.rb]: Why don't you used the source from the github repo?

File: packages/sysfsutils.rb (hunk 1 above)
Comment (JL2210) [reply packages/sysfsutils.rb]: If you don't mind the description being slightly long, I could do that.

File: packages/sysfsutils.rb (hunk 2 above)
Comment (JL2210) [reply packages/sysfsutils.rb]: The github repo doesn't have releases. Debian just packed the latest git source code into a tarball ( From 2010 )

Comment (JL2210) [reply packages/sysfsutils.rb]: How about a compromise? I got the source code from SourceForge.

File: packages/sysfsutils.rb (hunk 1 above)
Comment (JL2210) [reply packages/sysfsutils.rb]: Done.

**Pull request merged**